app/src/main/java/com/limelight/binding/input/KeyboardTranslator.java

```diff
@@ -24,6 +24,10 @@ public class KeyboardTranslator implements InputManager.InputDeviceListener {
     public static final int VK_0 = 48;
     public static final int VK_9 = 57;
     public static final int VK_A = 65;
+    public static final int VK_C = 67;
+    public static final int VK_D = 68;
+    public static final int VK_G = 71;
+    public static final int VK_V = 86;
```

Comment:
@kmreisi Where did you get the code "86" from?
I tried to built a version from your code few hours ago and I saw that VK_V=56 & VK_LCONTROL=11 worked for me. I have tested it from my device.
The key codes I got was from https://learn.microsoft.com/en-us/windows/win32/inputdev/virtual-key-codes
My code was like this
```java
                new MenuOption(ACTION_SEND_SPECIAL_KEYS_CTRL_V, () -> sendKeySequence(
                        (byte) 0, new short[]{0x11, 0x56})),
```
I have tried 
```java
                new MenuOption(ACTION_SEND_SPECIAL_KEYS_ESC, () -> sendKeySequence(
                        (byte) 0, new short[]{0x18})),

```
and it didn't work. Is there any chance that the your codes are incorrect?

Reply:
86 is decimal of 0x56 (hex). Since KeyboardTranslator.java is using decimal everywhere I aligned to this.

=> 0x56 is the same as 86. :-)

Could you please try my latest changes. (The delay is important!).

Of course there is always the chance for an issue, but I tried all the shortcut and the are working for me. :-)

Host is running latest Sunshine.

Reply:
@kmreisi new update:
I have used your latest changes. ESC mapping and other mappings worked fine except Ctrl+V.

![build](https://user-images.githubusercontent.com/5093683/213895883-9a702079-b1a1-49e7-a56d-87e9ace3d8f0.jpg)

With Ctrl+V, what I received is V.  So I temporarily used sendKeySequence to make it work. Not sure why it didn't work with sendKeys. Maybe the KEY_UP_DELAY affected this.

<img width="1090" alt="Screen Shot 2023-01-22 at 08 00 14" src="https://user-images.githubusercontent.com/5093683/213895858-05018d0f-dda4-4f31-996d-2ccbee02f4b2.png">

Reply:
@kmreisi Update: 
I can make it work by using this code
```java
    private void sendKeysWithCtrl(short[] keys) {
        for (short key : keys) {
            conn.sendKeyboardInput(key, KeyboardPacket.KEY_DOWN, KeyboardPacket.MODIFIER_CTRL);
        }

        new Handler().postDelayed((() -> {

            for (int pos = keys.length - 1; pos >= 0; pos--) {
                short key = keys[pos];

                conn.sendKeyboardInput(key, KeyboardPacket.KEY_UP, KeyboardPacket.MODIFIER_CTRL);
            }
        }), KEY_UP_DELAY);
    }

```
and then ...
```java
                new MenuOption(getString(R.string.game_menu_send_keys_ctrl_v),
                        () -> sendKeysWithCtrl(new short[]{KeyboardTranslator.VK_V})),

```

Reply:
@dangnm initially I have used the modifiers, since it has worked without for me I removed it to keep it a bit simpler.... :-)

With my latest patch  the sendKeys function should apply the modifiers correctly. Please try my latest changes again hopefully this works for GFE + Sunshine :-).

Reply:
Nice. It works really well now. Someone who is playing with a smart TV or TVbox may feel this PR very useful because now we can type texts in game by using an android phone with Swiftkey installed & clipboard sharing enabled in Windows. We just type in the phone, copy and then calling Ctrl+V.

Reply:
Thanks for the feedback. Nice to read its working now => resoled :-)